tests/test_supervisor.py

```diff
@@ -36,10 +36,7 @@ async def fixture_webession(coresys: CoreSys) -> AsyncMock:
 @pytest.fixture(name="supervisor_unthrottled")
 async def fixture_supervisor_unthrottled(coresys: CoreSys) -> Supervisor:
     """Get supervisor object with connectivity check throttle removed."""
-    with patch(
-        "supervisor.supervisor._check_connectivity_throttle_period",
```

Comment:
Yeah I think this really did not apply: The decorator is evaluated at import time and at that time this function is not yet patched. 

Tests did work in some cases because we only throttle after a successful connectivity check, but I was able to break things even with older pytest version.

Patching last_call on the Job decorator seems to work perfectly. :+1: